src/server/index_patterns/service/lib/index.js

```diff
@@ -1,3 +1,3 @@
 export { getFieldCapabilities } from './field_capabilities';
 export { resolveTimePattern } from './resolve_time_pattern';
-export { createNoMatchingIndicesError, isNoMatchingIndicesError } from './errors';
```

Comment:
Sorry, can you point me to where it's used? I grepped `src` for "isNoMatchingIndicesError" and didn't see any hits.